src/deepsparse/v2/text_generation/autoregressive_preprocess_operator.py

```diff
@@ -51,16 +50,19 @@ def can_operate(self, inp: Any) -> bool:
         tokens = inp.get("tokens")
         kv_cache = inp.get("kv_cache")
 
+        if inp.get("in_generation"):
```

Comment:
what is "in_generation"?

Reply:
flag to figure out if we're in the generation loop or prompt inference